src/ui/battle_ui.py

```diff
@@ -0,0 +1,58 @@
+import os
+from rich import print
+from rich.layout import Layout
+from rich.panel import Panel
+
+def construct_character_panel(fighter):
+  return Panel(fighter.name + '\n' +
+                'HP:'+ str(fighter.stats["hp"]-fighter.lost_hp) +'/' + str(fighter.stats["hp"]) + '\n' + 
+                'lvl:'+ str(fighter.level))
+
+def construct_character_row(fighters: [], layout: Layout,relation: str):
+    #TERRIBLE solution but no alternative was found :/
+    if (len(fighters)==1):
```

Comment:
I think you could have used the unpack (*) operator on lists. Something like:

layout[relation].split_row(*[construct_character_panel(fighter) for fighter in fighters])

But since we won't really have more than 4 enemies, this is fine too